src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,88 @@
+package duke.task;
+
+import duke.ui.Ui;
+
+import java.time.LocalDate;
+
+/**
+ * Handles the creation of tasks.
+ */
+public class Task {
+    private boolean marked;
```

Comment:
Consider renaming the booleans in this class?

Reply:
Good catch! Definitely missed this during my checks!